ActiveQuery.php

```diff
@@ -369,6 +366,21 @@ protected function executeScript($db, $type, $columnName = null)
      */
     private function findByPk($db, $type, $columnName = null)
     {
+        if (!empty($this->orderBy) && in_array($type, ['All', 'One', 'Count', 'Column'])) {
```

Comment:
Not need sorting for type 'Count'.
And this expression must be assigned to a variable because it's duplicated 2 times